### PR TITLE
feat: Improved canned response list design

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description 

- Improved canned response list rendering, like a pop view instead of scrolling up the chat screen.
- Minor chatlist [spacing](https://linear.app/chatwoot/issue/CW-1885/alignment-issues-with-the-thumbnail) and font weight fixes 

Fixes https://linear.app/chatwoot/issue/CW-1889/improve-the-canned-response-list-design , https://linear.app/chatwoot/issue/CW-1885/alignment-issues-with-the-thumbnail

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have tested in both Android and iOS platform
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by muhsin-k** [comment]
@iamsivin The UI is breaking if canned response is empty

Simulator Screenshot - iPhone 14 - 2023-06-19 at 11 23 00

**Comment by iamsivin** [comment]
> @iamsivin The UI is breaking if canned response is empty
> 
@muhsin-k Fixed

File: src/screens/ChatScreen/components/CannedResponses.js [hunk 1]
```diff
@@ -67,20 +82,23 @@ const CannedResponses = ({ cannedResponses, onClick }) => {
...
-            content={item.content}
-            lastItem={cannedResponses.length - 1 === index}
-            onClick={onClick}
-          />
-        )}
-        keyExtractor={item => item.id}
-        keyboardShouldPersistTaps={'handled'}
-      />
+    <View style={[cannedResponses.length > 0 && styles.mainView]}>
+      {cannedResponses.length > 0 && (
```
Comment (muhsin-k) [inline src/screens/ChatScreen/components/CannedResponses.js:86]: Create a flag variable and use it in JSX.
`const isCannnedResponsesExist = cannedResponses.length > 0 `

Comment (iamsivin) [reply src/screens/ChatScreen/components/CannedResponses.js]: Done

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**